test/function_test.py

```diff
@@ -467,12 +467,6 @@ def foo():
     assert function_id
     assert foo.web_url
 
-    rehydrated_function = Function.from_id(function_id, client=client)
```

Comment:
Confused about why this test is being removed — don't we still have `Function.from_id`?

Reply:
I dropped it from the base class in order to remove the need for `AppLookupObject`

Looking at internal dashboards and there is exactly zero backend calls in the last week. I see 1 (!) call in the last month. But it seems relatively safe to remove this for now and reintroduce if it's something users really really care about.

Reply:
Right, but we're not testing the base class here we're testing `Function` right?

Reply:
Yeah but I didn't drop the implementation from the base class to all subclasses – just to `FunctionCall` and `Sandbox`. The other classes lost the method. So `Function` doesn't have `from_id` anymore.

Reply:
Ah, missed the `Function`/`FunctionCall` distinction, thanks 👍